examples/pony/alerts_local_aggregations/alerts.pony

```diff
@@ -25,11 +25,13 @@ use "wallaroo/core/source/gen_source"
 use "wallaroo/core/state"
 use "wallaroo/core/topology"
 use "wallaroo/core/windows"
+use "wallaroo_labs/logging"
 use "wallaroo_labs/mort"
 use "wallaroo_labs/time"
 
 actor Main
   new create(env: Env) =>
+    Log.set_defaults()
```

Comment:
What happens for the calls to `@l` and `@ll` if this is not included in an application definition?

Reply:
1. All severity and category labels are set to defaults like `"sev-4"` and `"cat-22"`
2. The severity threshold for all categories is set to `MAX_SEVERITY` which is equivalent to `Log.debug()`